src/main/resources/swagger/api-docs.yaml

```diff
@@ -104,6 +104,120 @@ paths:
           description: Method Repository configuration detail
         500:
           description: Internal Server Error
+  /methods/{namespace}/{name}/{snapshotId}/permissions
```

Comment:
The swagger definitions should have model objects for post bodies and responses.